src/cos_integration.py

```diff
@@ -149,39 +139,43 @@ def get_metrics_endpoints(self, node_name: str, token: str, cluster_name: str) -
 
         kubelet_jobs = [
             JobConfig(
-                f"kubelet-{metric}" if metric else "kubelet",
-                path,
-                "https",
-                "localhost:10250",
-                [
-                    {"target_label": "metrics_path", "replacement": path},
+                name=f"kubelet-{metric.replace('/', '-')}",
+                metrics_path=metric,
+                scheme="https",
+                target="localhost:10250",
+                relabel_configs=[
```

Comment:
ok, does this mean that that is no longer want `JobConfig` for just `kubelet` at the `/metrics` path?

Seems to me this would generate this in the list of `kubelet-Jobs`

```python
JobConfig(
    name="kubelet-",
    metrics_path=""
    ...
```

Reply:
Alright, so the name is not important here. So the charm is using the relabel_configs, so all scraped  metrics from Kubelet will have `job="kubelet"`, no matter the actual job name. This is the expected behavior, as the alerts target the `job="kubelet"` label.

Reply:
neato